packages/components/src/styles/footnotes/footnotes.stories.ts

```diff
@@ -55,39 +52,42 @@ export const Default = {
 };
 
 /**
- * Use the class with a `ul` element to display the footnotes as an unordered list.
+ * Use the alternative html elements for alternative appearances:
+ *
+ * - unnumbered: use a `ul` element to create an unnumbered list.
```

Comment:
can we have numbered first? as it is the default?

Reply:
Updated